### PR TITLE
Create table deleted_bots

### PR DESCRIPTION
**Pull request by kulla**
After talking to @simonkoehl we noticed that we need to store deleted
bots since we want to remove them also from the newsletter later in the
futute. This PR creates a seperate table where we can store the data.

### CODE REVIEW

File: packages/migrations/src/202111012023800-create-table-deleted-bots.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { createMigration } from './utils'
+
+createMigration(exports, {
+  up: async (db) => {
+    await db.runSql(`CREATE TABLE IF NOT EXISTS deleted_bots LIKE user`)
+  },
+  // TODO: @inyono Do I need a down-type?!
+  /* down: async (db) => {
+    await db.runSql(`INSERT INTO type (name) VALUES ('text-hint')`)
+  },*/
```
Comment (kulla) [inline packages/migrations/src/202111012023800-create-table-deleted-bots.ts:31]: @inyono What is this down-command for? A possible undo for the migration?!

Comment (simonkoehl) [reply packages/migrations/src/202111012023800-create-table-deleted-bots.ts:31]: thanks, @kulla !

**Review by inyono — CHANGES_REQUESTED** [review]
Instead of persisting the deleted bots, why don't we directly remove them from the newsletter via Mailchimp API https://mailchimp.com/developer/marketing/api/list-members/delete-list-member/? Besides the id of the list, it only needs the (hashed) email address of the subscriber.

**Comment by kulla** [comment]
As discussed we will implement another solution...

**Pull request closed without merging**